configs/iterated-training/alternate-experiment.cfg

```diff
@@ -0,0 +1,7 @@
+# Victimplay config used on every other iteration of iterated training.
+@include ../active-experiment.cfg
+
+searchAlgorithm0 = AMCTS
```

Comment:
Maybe add a comment here denoting which one is the cyclic policy and which one is KataGo?